lib/active_model/entity/serializers/json.rb

```diff
@@ -7,8 +7,27 @@ module Serializers
       module JSON
         extend ActiveSupport::Concern
 
+        included do
+          class_attribute :custom_serializers, default: {}
+        end
+
         # Class-level methods.
         module ClassMethods
+          # Handle inheritance by clonning custom_serializers value
+          def inherited(subclass)
+            super
+
+            subclass.custom_serializers = custom_serializers.dup
+          end
+
+          #
+          # Specifies custom serialization for attribute.
+          # @param attribute [Symbol|String] The attribute to serialize.
+          # @param callable [Proc] The callable to use for serialization. Object or hash and options are passed as arguments.
+          def serialization(attribute, callable)
+            custom_serializers[attribute.to_s] = callable
+          end
```

Comment:
Can we:
1. Rename to `serializes` or `serialize_with` or smth like that?
2. Let's make `callable` a block? No point in adding more characters to express the same stuff

Reply:
👍